src/service/error/AbstractFileError.ts

```diff
@@ -0,0 +1,10 @@
+export abstract class AbstractFileError extends Error {
+    constructor(className: string,
```

Comment:
I do not think we need class name. In error handler we can use `err.constructor.name` where `err` is an instance of error. It should give us same result as `FileDownloadError.name` I suspect.

Reply:
I'll have a look at this